content/scalers/azure-storage-blob.md

```diff
@@ -0,0 +1,73 @@
++++
+fragment = "content"
+weight = 100
+title = "Azure Storage Blob"
+background = "light"
++++
+
+Scale applications based on Azure Storage Blobs.
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `azure-blob` trigger for Azure Storage Blob.
+
+```yaml
+triggers:
+  - type: azure-blob
+    metadata:
+      containerName: functions-blob # Required: Container name
+      blobCount: '5' # Optional. Blob count target for HPA. Default: 5 blobs 
+      connection: STORAGE_CONNECTIONSTRING_ENV_NAME
+      blobPrefix:  # Optional. Prefix for the Blob. Use this to specifiy sub path for the blobs if required. Default : ""
+      blobDelimiter: # Optional. Delimiter for identifying the blob Prefix. Default: "/"
+```
+
+The `connection` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
```

Comment:
I would leave this out as we now also support `TriggerAuthentication`

Reply:
Agreed. I'll first do the changes to work only with TriggerAuthentication and later remove this connection section.

Reply:
Ah now I get it - No, please keep this around as it should support both TriggerAuthentication & this approach.

Sorry for the confusion!

Reply:
Sure. No worries!